index.js

```diff
@@ -1,7 +1,50 @@
+
+
 for(let i = 0; i < document.querySelectorAll('button').length; i++){
-    document.querySelectorAll('button')[i].addEventListener('click', clicked);
+
+    document.querySelectorAll('button')[i].addEventListener('click', function (){
+      makeMatchingSound(this.innerHTML);
+      console.log(this.innerHTML)
+    });
 }
 
-function clicked (){
-    alert('I have been clicked');
+document.addEventListener('keydown',function(el){
```

Comment:
in future I should name this parameter a bit more clearly, its a bit vague as to what it is.